en/flight_controller/README.md

```diff
@@ -12,4 +12,38 @@ There are a number of options:
 * Commercial UAVs that can run PX4
   * [Crazyflie 2.0](../flight_controller/crazyflie2.md)
   * [Parrot Bebop](../flight_controller/bebop.md)
-  
\ No newline at end of file
+  
+  
+## Documented Boards
+
+The list of boards documented in this library can be found in the sidebar (reproduced below):
+
+> **Note** The library does not document all boards that can run PX4 (there are other compatible flight controllers and variants).
+
+* [Pixhawk Series](../flight_controller/pixhawk_series.md)
+  * [Pixhawk 1](../flight_controller/pixhawk.md)
+  * [mRo Pixhawk](../flight_controller/mro_pixhawk.md)
+  * [mRobotics-X2.1](../flight_controller/mro_x2.1.md)
+  * [HKPilot32](../flight_controller/HKPilot32.md)
+  * [Pixfalcon](../flight_controller/pixfalcon.md)
+  * [Dropix](../flight_controller/dropix.md) 
+  * [Pixracer](../flight_controller/pixracer.md)
+  * [MindPX](../flight_controller/mindpx.md)
+  * [MindRacer](../flight_controller/mindracer.md)
+    * [MindRacer BNF & RTF](../flight_controller/mindracer_BNF_RTF.md)
+      * [MindRacer 210](../flight_controller/mindracer210.md)
+      * [NanoMind 110](../flight_controller/nanomind110.md)
+  * [Pixhawk 2](../flight_controller/pixhawk-2.md)
+  * [Pixhawk Mini](../flight_controller/pixhawk_mini.md)
+  * [Pixhawk 3 Pro](../flight_controller/pixhawk3_pro.md)
+  * [Pixhack v3](../flight_controller/pixhack_v3.md)
+  * [AUAV-X2 (Discontinued)](../flight_controller/auav_x2.md) 
+  * [Snapdragon Flight](../flight_controller/snapdragon_flight.md)
```

Comment:
This is indented too much, Snapdragon is not part of the Pixhawk Series.

Reply:
Thanks! Fixed in https://github.com/PX4/px4_user_guide/commit/e8193ba40fef4ff37893d37fc2e437e279e77983